classes/class-email-service.php

```diff
@@ -178,22 +187,47 @@ public function action( $action, Backup $backup ) {
 					$error_message = ' - ' . $error_message;
 				}
 
-				$subject = sprintf( __( 'Backup of %s Failed', 'backupwordpress' ), $domain );
-
-				$message = sprintf( __( 'BackUpWordPress was unable to backup your site %1$s.', 'backupwordpress' ) . "\n\n" . __( 'Here are the errors that we\'ve encountered:', 'backupwordpress' ) . "\n\n" . '%2$s' . "\n\n" . __( 'If the errors above look like Martian, forward this email to %3$s and we\'ll take a look', 'backupwordpress' ) . "\n\n" . __( "Kind Regards,\nThe Apologetic BackUpWordPress Backup Emailing Robot", 'backupwordpress' ), home_url(), $error_message, 'backupwordpress@hmn.md' );
+				$subject = sprintf(
+					/* translators: Domain URL */
+					esc_html__( 'Backup of %s Failed', 'backupwordpress' ),
+					$domain
+				);
```

Comment:
This is an email subject, so instead of escaping, `wp_strip_all_tags()` should be used.